include/zephyr/bluetooth/gatt.h

```diff
@@ -1777,9 +1777,21 @@ enum {
 	/** @brief Write pending flag
 	 *
 	 *  If set, indicates write operation is pending waiting remote end to
-	 *  respond.
+	 *  respond. Internal use only.
 	 */
-	BT_GATT_SUBSCRIBE_FLAG_WRITE_PENDING,
+	BT_GATT_SUBSCRIBE_INTERNAL_FLAG_WRITE_PENDING,
```

Comment:
Renaming this is a stable API change, and would probably need to follow the process in https://docs.zephyrproject.org/latest/develop/api/api_lifecycle.html#stable. 

I would suggest to leave the name as is, and keep the comment about it being internally used too.

Reply:
don't really thing it's an API change since it was not written anywhere that this was a public flag.
I still see your point that technically it is though.

Reply:
> don't really thing it's an API change since it was not written anywhere that this was a public flag.

Generally I'd assume that everything in `include/zephyr` is public, and before this PR it wasn't specified that this was an internal flag :P